Drafts/1NNN-VLP.md

```diff
@@ -0,0 +1,65 @@
+# Aliases of member functions in template arguments
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Victor Porton porton@narod.ru                                   |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Draft                                                           |
+
+## Abstract
+
+Member functions should be passeable as alias template arguments.
+
+## Contents
+* [Rationale](#rationale)
+* [Description](#description)
+* [Copyright & License](#copyright--license)
+* [Reviews](#reviews)
+
+## Rationale
+
+"Regular" aliases can be assigned to a member function like this:
+
+```d
+struct S {
+  void f() { }
+}
+S s;
+alias f = s.f;
+```
+
+But it does not work for template arguments:
+
+```d
+import std.stdio, std.meta;
+
+struct S {
+    void f() { }
+}
+
+void t(alias f)() { f(); writeln ("hello world"); }
+void main()
+{
+    S s;
+    t!(s.f); // does not compile
+}
+```
+
+## Description
+
+I propose to allow member functions as template alias arguments.
+
+The benefits of this change are obvious.
```

Comment:
I think if a DIP is required, it's not obvious. The pros & cons of this vs [dukc's comment](https://github.com/dlang/DIPs/pull/144#issuecomment-469869768) should be discussed in the DIP.